actionRuntimes/ruby2.5Action/build.gradle

```diff
@@ -0,0 +1,2 @@
+ext.dockerImageName = 'action-ruby-v2.5'
```

Comment:
And this same comment as dockerfile.

Reply:
Got it🙋‍

Reply:
Same comment as [this](https://github.com/apache/incubator-openwhisk/pull/3725#discussion_r196755418) applies here